python/bruninga/ax25_fsk_mod.py

```diff
@@ -76,7 +76,8 @@ def handle_msg(self, msg_pmt):
             return
         
         try:
-            msg = pickle.loads(msg[1])
+            msg = pickle.loads(bytes(msg[1]))
+            print("info field:", msg.info)
```

Comment:
Is this `print` statement meant to be added here, or were you just using it for testing?

Reply:
good catch! I forgot that one